### ISSUE TITLE
Importing *.graphql files via absolute path does not work

### ISSUE THREAD

**Issue by RemLampa** [apollographql/graphql-tag#221]
**Issue Labels**

- [x] has-reproduction
- [ ] feature
- [ ] docs
- [ ] blocking
- [ ] good first issue

In my webpack config: `resolve.modules = [ 'src', 'node_modules'];`

I have various folders inside `src`, notably `components` and `graphql-docs`.

This absolute import works: `import SomeComponent from 'components/SomeComponent';`

This relative import also works: `import Query from './graphql-docs/SomeQuery.graphql';`

But this absolute import doesn't work: `import Query from 'graphql-docs/SomeQuery.graphql';`

```
Error: /path/to/src/AnotherComponent.jsx: Cannot find module 'graphql-docs/SomeQuery.graphql'
```

This is somewhat related to #198, but that points out a bug in Webpack aliases, while this one is about the Webpack module resolver.

**Comment by mhaagens** [comment]
Samme issue here.

**Comment by langpavel** [comment]
Yeah, this is probably related to webpack (may be in your config) itself, not this loader.
Try add loader as first in chain.

**Comment by nazar** [comment]
>Yeah, this is probably related to webpack (may be in your config) itself, not this loader.
Try add loader as first in chain.

Nope:

```
    module: {
      rules: [
        {
          test: /\.(graphql|gql)$/,
          exclude: /node_modules/,
          use: 'graphql-tag/loader',
        },
    }
```

Throws a `GraphQLError: Syntax Error: Unexpected Name "module"` error on the first .gql the loader encounters in a CRA 2.x ejected application.

Sorry @langpavel but your comment is cryptic. I would be most grateful if you would elaborate 🙏

**Comment by dobesv** [comment]
I don't think this is a bug in `graphql-tag`, maybe you should try stack overflow or somewhere people can help you with `webpack` questions.
